src/layouts/contexts/WhaleContext.tsx

```diff
@@ -101,3 +117,18 @@ function newRpcClient(options: WhaleApiClientOptions): WhaleRpcClient {
     `${options.url}/${options.version}/${options.network}/rpc`
   );
 }
+
+export function newPlaygroundClient(
```

Comment:
These are already existing on walletkit, let's reuse that

Reply:
https://github.com/WavesHQ/walletkit/pull/34

Reply:
@thedoublejay we will be removing this set of code when OCG will be available on mainnet/testnet, This is just for testing purpose